scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model and Concepts
+
+This topic describes data models used in the Supply Chain Security Tools (SCST) – Artifact Metadata Repository (AMR).
+
+## <a id='overview'></a> Overview
+
+The diagram below shows the data models used in the AMR to store artifact information and the relation between them.
+
+![Diagram of the AMR Data Models](../images/amr-data-model.jpg)
+
+## AMR Data Models
+
+There are two types of data models in the AMR, `Apps` and `Locations`.
+
+### Apps
+
+The `Apps` data model represents the information of an application (i.e. replicaset) at a point in time.
+This information is sent to the AMR by [cloud events](cloudevents.hbs.md), so each entry represents the status of the app when an event occurred.
+These entries correspond to events such as when an app is created, scaled up, running, scaled down or deleted.
+This is useful for users to see the history of their apps.
+
+Each `Apps` data entry stores information on the associated location it was deployed in, details on the status, what is deployed etc. Check the diagram above for a list of all fields the data model stores.
+An `Apps` entry can only be associated to one `Locations` entry. 
+There can be multiple `Apps` entries pointing to the same `Locations` entry, but no duplicates are allowed in the same location.
+
+### Locations
+
+The `Locations` data model stores data about locations that `Apps` are deployed on (i.e. clusters).
```

Comment:
Would it make sense to list the specific fields/data that gets stored? And then for each field, add a bit of a description.

Reply:
Since the fields are already listed in the diagram, I thought it'd be a little redundant to list it here again. Do you think it's still useful to add it here?

Reply:
You list them directly below, but in a "wall of text" type format. My thought process is that a bulleted list would be easier for a lazy reader to digest.

Reply:
Ah I see, okay I changed it to list format